benchmarks/benchmark_serving.py

```diff
@@ -297,17 +297,41 @@ def sample_random_requests(
 async def get_request(
     input_requests: List[Tuple[str, int, int]],
     request_rate: float,
+    burstiness: float,
 ) -> AsyncGenerator[Tuple[str, int, int], None]:
+    """
+    Asynchronously generates requests at a specified rate 
+    with OPTIONAL burstines.
```

Comment:
```suggestion
    with OPTIONAL burstiness.
```

Reply:
Thanks! The format check has been passed now.